galsim_extra/mixed_scene.py

```diff
@@ -58,8 +59,23 @@ def setup(self, config, base, xsize, ysize, ignore, logger):
         # object ends up being chosen.
         ignore = ignore + ['objects', 'magnify', 'shear', 'obj_type']
 
+        stamp_xsize, stamp_ysize, image_pos, world_pos = super(MixedSceneBuilder, self).setup(config,base,xsize,ysize,ignore,logger)
+        # option to shear the full scene.
+        if config.get('shear_full_scene', True):            
```

Comment:
Definitely don't make the default True.  This will break everyone else who uses this type and doesn't want this new behavior change.

Also, don't use get.  This should be done using the galsim value parsing so the bool value can be a computed value (e.g. different for different exposures/objects/whatever).
```
if 'shear_full_scene' in config:
    shear_full_scene = galsim.config.ParseValue(config, 'shear_full_scene', base, bool)[0]
else:
   shear_full_scene = False
if shear_full_scene:
```